ons-spark/raw-notebooks/checkpoint-staging/r_input.R

```diff
@@ -0,0 +1,155 @@
+options(warn = -1) 
+library(sparklyr)
+library(dplyr)
+library(DBI)
+
+sc <- sparklyr::spark_connect(
+    master = "local[2]",
+    app_name = "checkpoint",
+    config = sparklyr::spark_config())
+
+
+set.seed(42)
+new_cols <- 12
+num_rows <- 10^3
+
+start_time <- Sys.time()
+
+df = sparklyr::sdf_seq(sc, 1, num_rows) %>%
+    sparklyr::mutate(col_0 = ceiling(rand()*new_cols))
+
+for (i in 1: new_cols)
+{
+  column_name = paste0('col_', i)
```

Comment:
Minor stylistic point - hopefully not too pedantic but there are a few instances where you use = instead of <-, for example on lines 23, 24, 38 and 84